package.json

```diff
@@ -1,6 +1,7 @@
 {
-  "version": "0.1.3-rc.4",
+  "version": "0.2.0-alpha.2",
```

Comment:
maybe we should remove version from here at all? we don't publish this package json and don't update version here

Reply:
This value is used in workspace init script, to get current gene packages